api/app/settings/common.py

```diff
@@ -199,6 +199,7 @@
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
     "simple_history.middleware.HistoryRequestMiddleware",
+    "core.middleware.cache_control.NeverCacheMiddleware",
```

Comment:
Should we make this configurable perhaps? Just wondering if self hosted people might be implementing some caching.